pbjson-types/src/lib.rs

```diff
@@ -36,6 +37,10 @@ mod pb {
 }
 
 mod duration;
+mod list_value;
+mod null_value;
+mod r#struct;
 mod timestamp;
+pub mod value;
```

Comment:
I think I would prefer to keep the module topology flat, currently all inner modules are private and the types are re-exported by the `pub use pb::google::protobuf::*` a few lines down

`value` does not appear to define any new public types, but just define some trait impls for `crate::Value`. Is there a particular reason you wish to have a value module?

Reply:
>  Is there a particular reason you wish to have a value module?

If I was looking for the implementation of `Value`, intuitively I'd look for `src/value.rs`, the topology isn't flat because `pb::google::protobuf::*` already exports a `value` module. So this makes so the actual code structure reflect what people see in the documentation, and removes the conflicts that arise from using glob exports when it is private. `mod value` overrides the `value` from `pb::google::protobuf::*`, but this wouldn't be obvious to you unless you removed the `pub mod` and `pub use pb::protobuf::google::protobuf::Kind` and then tried to use `crate::value::Kind`).

Reply:
Oh, yes prost is generating a `value` namespace within `pb::google::protobuf` because `Kind` is defined within the `Value` message. Gotcha, that is definitely confusing but makes sense :+1: